mteb/tasks/Classification/eng/ToxicChatClassification.py

```diff
@@ -81,14 +51,17 @@ class ToxicChatClassification(AbsTaskClassification):
     )
 
     def dataset_transform(self):
-        # only use human-annotated data
-        self.dataset = self.dataset["test"].filter(lambda x: x["human_annotation"])
         keep_cols = ["user_input", "toxicity"]
         rename_dict = dict(zip(keep_cols, ["text", "label"]))
-        remove_cols = [col for col in self.dataset.column_names if col not in keep_cols]
+        remove_cols = [
+            col
+            for col in self.dataset[_EVAL_SPLITS[0]].column_names
```

Comment:
the info about splits is in metadata by this point, why referening private global variable here with arbitrary index 0?